static/script/devices/mediaplayer/html5.js

```diff
@@ -587,7 +587,7 @@ require.def(
                         self._mediaElement.currentTime = self._sentinelSeekTime;
                     });
                 } else {
-                    this._sentinelSeekTime = currentTime;
+                    this._sentinelSeekTime = undefined;
```

Comment:
Well, my thinking was that the seek sentinel had done its job at this point, so it could safely be disabled until the next playFrom(). But we know that some devices report the target seek position as the current time, then hop back to the original time, so this won't actually work.

Since this pull request hasn't broken any unit tests, we obviously have a gap in our testing. I'm going to add a unit test for the seek-then-hop-back scenario before going any further.